Baconeta-Studios/setting-the-stage
# Change spotlight selection

closes #90 
progress #74 

- Stage Selection can now swap between positions with arrows on the left and right.
- Can no longer click on other stage positions to swap to them.
- Stage Selection now shows both Musician and Instrument carousels.

Testing:
1. Start in Act 1
2. Enter Chapter 
3. Click Intro to advance to Stage Selection
4. Click on a Stage Position
5. Both carousels should show, and the camera zooms in.
6. Clicking a stage position does nothing
7. Clicking the Left/Right button changes the stage position to the left or right, with wrapping.
